chapter1/challenges/program/lib/password_validator.rb

```diff
@@ -3,5 +3,14 @@
 # - include at least one of these characters: !, @, $, %, &
 
 def valid?(password)
+  if (password.length < 8)
+    return false
+  else
```

Comment:
It's a matter of preference, but I find elsif to be a nicer construct since it makes the number of blocks seem less. In fact: since the if block returns when the condition is true, you could have closed that block and just started another if.